packages/trpc/src/procedures/user.ts

```diff
@@ -0,0 +1,6 @@
+import { authenticatedProcedureBuilder } from "../internal/init";
+
+export const getAuthenticatedUserProcedure =
+  authenticatedProcedureBuilder.query(({ ctx }) => {
```

Comment:
Let's
1) Verify that this endpoint doesn't return the user's password
2) Write a test specifically for that behavior